pygenometracks/tracks/BedGraphTrack.py

```diff
@@ -222,7 +224,10 @@ def plot(self, ax, chrom_region, start_region, end_region):
             score_list, x_values = self.get_values_as_bdg(score_list,
                                                           pos_list)
 
-        plot_coverage(ax, x_values, score_list, self.plot_type, self.size,
+        new_scores = transform(score_list, self.properties['transform'],
```

Comment:
what about transformed_scores instead of new_scores? just to be more informative with the naming.

Reply:
You are right, I will change it.